packages/isomorphic/README.md

```diff
@@ -0,0 +1,102 @@
+# @xrplf/isomorphic
+
+A collection of isomorphic implementations of crypto and utility functions.
+
+Browser implementations of cryptographic functions use `@noble/hashes` and `crypto` for node .
+
+### Hashes
+
+All hash functions (similar to `@noble/hashes`):
+
+- can be called directly, with Uint8Array.
+- return Uint8Array
+- can receive string, which is automatically converted to Uint8Array via utf8 encoding (not hex)
+
+```
+function hash(message: Uint8Array | string): Uint8Array;
+hash(new Uint8Array([1, 3]));
+hash('string') == hash(new TextEncoder().encode('string'));
+```
+
+All hash functions can be constructed via `hash.create()` method:
+
+- the result is `Hash` subclass instance, which has `update()` and `digest()` methods
```

Comment:
I'd recommend capitalizing every bullet and making them complete sentences for the README.md
```suggestion
- The result is `Hash` subclass instance, which has `update()` and `digest()` methods.
```